### DISCUSSION TITLE
How to install multiple themes on website?

### DISCUSSION THREAD

**Discussion by HuakunShen** [skeletonlabs/skeleton#1796] [Q&A]
How to install multiple themes like the documentation website and be able to toggle themes?

Should I import all css files? Or should I import the css files dynamically?

```ts
import '@skeletonlabs/skeleton/themes/theme-skeleton.css';
// import '@skeletonlabs/skeleton/themes/theme-crimson.css';
// import '@skeletonlabs/skeleton/themes/theme-gold-nouveau.css';
// import '@skeletonlabs/skeleton/themes/theme-hamlindigo.css';
// import '@skeletonlabs/skeleton/themes/theme-modern.css';
// import '@skeletonlabs/skeleton/themes/theme-rocket.css';
// import '@skeletonlabs/skeleton/themes/theme-sahara.css';
// import '@skeletonlabs/skeleton/themes/theme-seafoam.css';
// import '@skeletonlabs/skeleton/themes/theme-seasonal.css';
// import '@skeletonlabs/skeleton/themes/theme-vintage.css';
// If you have source.organizeImports set to true in VSCode, then it will auto change this ordering
import '@skeletonlabs/skeleton/styles/skeleton.css';
// import '@skeletonlabs/skeleton/styles/sahara.css';
// import '@skeletonlabs/skeleton/styles/seafoam.css';
```

**Comment by endigo9740** [comment]
@HuakunShen we do not currently provide support for multiple themes in Skeleton due to the complexity required. You can view how we handle it in our doc site since it's open source, but it requires several steps. That said, this will change in the upcoming v2 release. We've reworked our Tailwind plugin in such a way this is much more trivial. Essentially you register your themes and then swap a data attribute on the body element to change themes. You cand find details here:

- https://github.com/skeletonlabs/skeleton/issues/1350

**Reply by HuakunShen** [reply]
The new design should be great, like daisyui. Their theme change only requires an attribute change.

Can you point me to the file in your doc site that this is used? I couldn't find it.

Is there a estimated time for the v2 release. We really need this feature.

**Reply by endigo9740** [reply]
Documentation will be written soon, and no set ETA for release. Likely the one release or one after though. We typically release on a 2 week interval, but this next release will be delayed a week since I'm out of office this week.

You can track progress for remaining items here:
- https://github.com/skeletonlabs/skeleton/discussions/1367

**Reply by HuakunShen** [reply]
Thank you for your work!
